python/lsst/ts/aos/analysis/nightlyAnalyzer.py

```diff
@@ -562,12 +597,152 @@ def _set_xticks(vals: pd.DataFrame, ax: plt.Axes) -> None:
+            The axes to plot on
+        data : pandas.DataFrame
+            The data to plot
+
+        Returns:
+        --------
+        ax_alt : matplotlib.axes.Axes
+            The secondary y-axis for altitude
+        """
+
+        # Plot rotator position and elevation
+        rot = data.groupby("seq")["rotation"].median()
+        alt = data.groupby("seq")["altitude"].median()
+
+        ax.set_ylabel('Rotation')
+        ax.plot(
+            rot.values,
+            c="blue",
+            alpha=0.5,
+            lw=1,
+            label="Rotator",
+        )
+        ax.axhline(0., c="C2", ls=":")
+        ax.tick_params(axis='y')
+
+        # Create second y-axis that shares the same x-axis
+        ax_alt = ax.twinx()
+
+        ax_alt.set_ylabel('Altitude')
+        ax_alt.plot(
+            alt.values,
+            c="red",
+            alpha=0.5,
+            lw=1,
+            label="Altitude",
+        )
+        ax_alt.tick_params(axis='y')
+
+        # Create legend
+        handles, labels = ax.get_legend_handles_labels()
+        handles2, labels2 = ax_alt.get_legend_handles_labels()
+        handles += handles2
+        labels += labels2
+        ax.legend(
+            handles, labels,
+            bbox_to_anchor=(0, 1.1, 1, 0.13),
+            loc="upper center",
+            borderaxespad=0,
+            ncol=2,
+        )
+
+        return ax_alt
+
+    def _plot_temperature(self, ax: plt.Axes, data: pd.DataFrame) -> plt.Axes:
+        """
+        Plot temperature data.
+
+        Parameters:
+        -----------
+        ax : matplotlib.axes.Axes
+            The axes to plot on
+        data : pandas.DataFrame
+            The data to plot
+
+        Returns:
+        --------
+        ax : matplotlib.axes.Axes
+            The axes with the temperature plot
+        """
+        # Plot temperature difference
+        tempAbove = data.groupby("seq")["above_glass_temperature"].median()
+        tempBelow = data.groupby("seq")["glass_temperature"].median()
+        deltaT = tempAbove - tempBelow
+
+        ax.set_ylabel('T')
+        ax.plot(
+            tempAbove.values,
+            c="C0",
+            alpha=0.5,
+            lw=1,
+            label="Above Glass Temp",
+        )
+        ax.plot(
+            tempBelow.values,
+            c="C0",
+            alpha=0.5,
+            lw=1,
+            ls='-.',
+            label="Glass Temp",
+        )
+        ax.set_ylim(
+            min(tempBelow.min(), tempAbove.min()) - 1,
+            max(tempBelow.max(), tempAbove.max()) + 1
+        )
+        ax.tick_params(axis='y')
+
+        # Create second y-axis that shares the same x-axis
+        ax_delta_t = ax.twinx()
+        ax_delta_t.set_ylabel('$\\Delta$ T')
+        ax_delta_t.plot(
+            deltaT.values,
+            c="C1",
+            alpha=0.5,
+            lw=1,
+            ls='--',
+            label="Temp Diff",
+        )
+        ax_delta_t.axhline(0., c="k", ls=":")
```

Comment:
ok, maybe that's not needed, since we don't really expect temperature to be necessarily around 0 (it's not like AOS FWHM)... And since it's not in legend, I'd remove that

Reply:
I'd opt for just adding it to the legend so we're not guessing what it is